README.md

```diff
@@ -164,6 +164,10 @@ Verdaccio aims to support all features of a standard npm client that make sense
 
 - npm audit - **supported**
 
+## Report a vulnerability
+
+If you want to report a security vulnerability, please follow the steps we have defined for you in our [security policy](https://github.com/verdaccio/verdaccio/security/policy).
```

Comment:
the steps we => the steps that we
Or "which" instead of "that". Sounds a bit better with that / switch.

Rest is ok =)